src/storage/txn/actions/prewrite.rs

```diff
@@ -358,12 +364,15 @@ impl<'a> PrewriteMutation<'a> {
         }
     }
 
+    fn need_min_commit_ts(&self) -> bool {
```

Comment:
Seems we are using different ways to judge if the current prewrite processing should use async commit logic, like 
`let need_min_commit_ts = secondaries.is_some() || self.try_one_pc;` which uses the `secondaries` field in the `prewriter`, and we've got some fields like `CommitKind`. Do we have methods to remove or reduce some fields?

Reply:
I'm not sure if we have time to do this before the 5.0 GA as it may introduce extra risks, maybe we could have a try later.

Reply:
Agree. I tried to unify the judgment, but seems it's a little dangerous to do it for now.